### ISSUE TITLE
Does not recognize empty categories

### ISSUE THREAD

**Issue by mahyar** [drkameleon/iswift#22]
For example:

```
@interface JournalTabViewController () //error on this line, unexpected (
@property (nonatomic) UIViewController *hi;
@end

@implementation JournalTabViewController

- (void)viewDidLoad {
    [super viewDidLoad];
}

@end
````

**Comment by drkameleon** [comment]
Well, I can confirm this. They're currently not recognized though they are in my to-do list and I'll my best to get it into the very next - 1.4 - release. :)

**Comment by drkameleon** [comment]
Fixed as of the upcoming 1.4. ;-)

------

**Example:**

```objective-c
@interface JournalTabViewController () //error on this line, unexpected (
@property (nonatomic) UIViewController *hi;
@end

@implementation JournalTabViewController

- (void)viewDidLoad {
    [super viewDidLoad];
}

@end
```

**Outputs:**

```swift
class JournalTabViewController {

	var hi: UIViewController?
	
	func viewDidLoad() {
		super.viewDidLoad()
	}	
}
```
